spec/inputs/tcp_spec.rb

```diff
@@ -23,13 +24,14 @@
       plugin = LogStash::Inputs::Tcp.new("codec" => LogStash::Codecs::JSON.new, "port" => 0)
       plugin.register
       insist { plugin.codec }.is_a?(LogStash::Codecs::JSONLines)
+      plugin.teardown
     end
   end
 
-  describe "read plain with unicode", :socket => true do
+  it "should read plain with unicode" do
```

Comment:
I tend to avoid to use of should in test example naming... a good explanation of why http://betterspecs.org/#should